README.md

```diff
@@ -1216,6 +1220,79 @@ const Select = ({
 };
 ```
 
+### 7.9 Conditional Rendering
+
+#### 7.9.1 Using short circuit
+
+when we only need to validate a logical case and return a component, we can directly use the short circuit
+
+**✅ Good:**
+
+```tsx
+import { useState } from 'react'
+import Welcome from '../components/Welcome'
+
+const HomePage = () => {
+  const [showWelcome, setShowWelcome] = useState(true)
+
+  return (
+    {showWelcome && <Welcome />}
+  );
+};
+```
+
+**❌ Bad:**
+
+```tsx
+import { useState } from 'react'
+import Welcome from '../components/Welcome'
+
+const HomePage = () => {
+  const [showWelcome, setShowWelcome] = useState(true)
+
+  return (
+    {showWelcome ? <Welcome /> : <></>}
```

Comment:
I understand that in the case of ternaries with empty return, the use of a short circuit makes it more readable

It's the pattern we adopt in most cases in our React projects

Reply:
Eu criei uma thread no slack: https://juntossomosmais.slack.com/archives/CLP6B13J5/p1677166436778949